packages/patternfly-4/react-table/src/components/Table/Table.d.ts

```diff
@@ -58,6 +58,8 @@ export interface TableProps extends Omit<Omit<HTMLProps<HTMLTableElement>, 'onSe
   onSelect?: Function;
   onSort?: Function;
   actions?: Array<IAction | ISeparator>;
+  actionResolver?: (rowData: Object, extraParams: Object) => Array<IAction | ISeparator>;
+  areActionsDisabled?: (rowData: Object, extraParams: Object) => boolean;
```

Comment:
yes - this resolves my other issue, allow rows to disable certain actions on a row by row basis. Did not realize `rowIndex` was included in the callback initially.